lib/policyEvaluator/RequestContext.ts

```diff
@@ -36,6 +36,9 @@ function _findAction(service: string, method: string) {
             return actionMapSTS[method];
         case 'metadata':
             return actionMapMetadata[method];
+        case 'scuba':
+            // currently only method is GetMetrics
+            return `scuba:${method}`;
```

Comment:
Let's create an action map for scuba, even if we only have one action for now, what do you think?

Reply:
Done, but now everytime we want to add an action in scuba we will have to also update the mapping in arsenal